includes/spoolchanges.js

```diff
@@ -71,18 +67,23 @@ module.exports = function(dbUrl, log, bufferSize, callback) {
     url: dbUrl + '/_changes',
     qs: { seq_interval: 10000 }
   };
-  client(r)
-    .pipe(liner())
+  var c = client(r);
+  c.end();
+
+  c.on('response', function(resp) {
+    if (resp.statusCode !== 200) {
+      c.abort();
+      callback(new error.BackupError('SpoolChangesError', 'Changes request failed with status code ' + resp.statusCode));
+    } else {
+      console.error('Streaming changes to disk:');
+    }
+  }).pipe(liner())
     .pipe(change(onChange))
     .on('finish', function() {
       processBuffer(true);
       logStream.write(':changes_complete ' + lastSeq + '\n');
       logStream.end();
       console.error('');
-      if (doccount === 0) {
-        callback(new Error('zero documents found - nothing to do'), null);
-      } else {
-        callback(null, {batches: batch, docs: doccount});
-      }
+      callback();
```

Comment:
Do we still need the `doccount === 0` check? I guess the only case where this can now happen is on an empty database. Even so it should probably just print a warning rather than throwing an error.